### DISCUSSION TITLE
[Feature] convert live photo to motion photo

### DISCUSSION THREAD

**Discussion by aviv926** [immich-app/immich#9209] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

A great feature that could be added to Immich would be to convert the `HEIC` files to a motion photo jpg.
I can see many advantages in this:

* Most computers do not support the `HEIC` extension natively to display images.
* The image still remains as a live photo only in a different format.
* There is a [ready script that can do this job.](https://github.com/mihir-io/MotionPhotoMuxer/tree/main)
* It is convenient to have one file than an image file and a video file.
* Most of the cloud services export the file as jpg + mov. If we add this option it can provide an advantage if a user wants to upload the files to another image storage service in the future.

My use case of this:
If anyone ever uses the nightmare called google takeout and has iPhone live photos and wants to import them to Immich/another platform and still keep the live file as live use [this amazing tool](https://github.com/mihir-io/MotionPhotoMuxer/tree/main) and you will get one jpg file (motion photo) it saved me so much headache . After searching for days how to import thousands of such images without problems from google takeout.

It's a bit counterintuitive that Immich never touches the user's file, but maybe we could allow the user to choose?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by PANICBUTT0N** [comment]
Alternatively, Immich could continue to store live photos and just convert them to motion photos on the fly when downloaded from the Android app. This way, Android devices can still download live photos as motion photos without Immich needing to write changes to live photo files (e.g. if you have live photos stored in a read-only external library).

**Reply by aviv926** [reply]
This is a good idea, maybe it will be an option when
[Real-time transcoding/streaming](https://discord.com/channels/979116623879368755/1327726696421523486) is implemented
